packages/default-storage/package.json

```diff
@@ -23,6 +23,7 @@
     "Evan Bacon <bacon@expo.io> (https://github.com/evanbacon)",
     "Tommy Nguyen <4123478+tido64@users.noreply.github.com> (https://github.com/tido64)"
   ],
+  "source": "src/index",
```

Comment:
Out of curiosity, which tool is using this field?

Reply:
I found this in Bob, required to correctly find entry point to the example app